package/cfg/k3s-cis-1.24-permissive/master.yaml

```diff
@@ -134,7 +134,7 @@ groups:
         scored: false
 
       - id: 1.1.10
-        text: "Ensure that the Container Network Interface file ownership is set to root:root (Automated)"
+        text: "Ensure that the Container Network Interface file ownership is set to root:root (Manual)"
         type: "skip"
```

Comment:
This links to the other question. Would skipped tests not be marked as "Automated"?